File management system/Student File Management System/Student File Management System/School.cs

```diff
@@ -12,5 +13,20 @@ public class School
          List<Student> students = new List<Student>();
 
         internal List<Student> Students { get => students; set => students = value; }
+        public int getIndexOfStudent(School school, int studentID) {
```

Comment:
this method shouldnt be in the school.cs class. it should be in the student classes because its more related to student than school